crates/sncast/tests/data/scripts/declare/test_scripts/Scarb.toml

```diff
@@ -0,0 +1,19 @@
+[package]
```

Comment:
nit: test_scripts as dir name is not very descriptive

Reply:
without `missing_field` folder, there will be no need for `test_scripts`

Reply:
good idea

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1705/commits/0fd458b6376af37b2b9150aed0ca47b634c764ec

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1705/commits/0a4d1a9007a34622c242a6dca1a1d6122b599c15